### PR TITLE
iframe carousel feature

### PR DESCRIPTION
**Pull request by ytyeoh**
css for iframe carousel

### CODE REVIEW

File: vendor/assets/stylesheets/reviews/_iframe.scss [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+.image img{
+	width: 100%;
+}
+.red {
+	border: 3px solid #FF0000;
+}
+
+/*share font*/
+.red-color{
+	color: red;
```
Comment (AriusLee) [inline vendor/assets/stylesheets/reviews/_iframe.scss]: Use $primary-color for favful red color. You can check the color var in colors.scss

File: vendor/assets/stylesheets/reviews/_iframe.scss [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+}
+.image{
+	padding: 0;
+}
+
+.image img{
+	width: 100%;
+}
+.red {
+	border: 3px solid #FF0000;
```
Comment (AriusLee) [inline vendor/assets/stylesheets/reviews/_iframe.scss]: Don't use so general classname like .image or .red etc, unless you want make it a shared general style. Then they should be put in respective stylesheet such as colors.scss, typo.scss, buttons.scss etc.

File: vendor/assets/stylesheets/reviews/_iframe.scss [hunk 3]
```diff
@@ -24,10 +24,10 @@
 .red-color{
 	color: red;
 }
+a.red-color:hover,a.red-color:active
```
Comment (AriusLee) [inline vendor/assets/stylesheets/reviews/_iframe.scss]: New line for sibling properties. i.e. new line after ","

File: vendor/assets/stylesheets/reviews/_iframe.scss [hunk 4]
```diff
@@ -17,22 +18,34 @@
...
-	color: red;
+a.red-color:hover,a.red-color:active,a.red-color:visited,a.red-color:focus{
+	color: #ED145B;
 	text-decoration: none;
 }
+.down{
+	position: absolute;
+	right: 5px;
+	bottom: 5px;
+}
```
Comment (AriusLee) [inline vendor/assets/stylesheets/reviews/_iframe.scss]: As mentioned in my previous comment, don't use so general classname for so specific styles like this.

File: vendor/assets/stylesheets/reviews/_iframe.scss [hunk 5]
```diff
@@ -8,6 +8,7 @@
     font-weight: 700;
     color: #000000;
     margin-bottom: 15px;
+    margin-top: 20px;
 }
 .image{
```
Comment (AriusLee) [inline vendor/assets/stylesheets/reviews/_iframe.scss]: Space between property and "{"

**Comment by AriusLee** [comment]
Check your editor's indentation setting before continue on working anything else. It should be just "2 spaces". Then fix to tab indent to 2 space indent, because it will bring alot code conflicts to others that using editor like vim and emac.

File: vendor/assets/stylesheets/reviews/_iframe.scss (hunk 1 above)
Comment (ytyeoh) [inline vendor/assets/stylesheets/reviews/_iframe.scss]: change with $primary-color

File: vendor/assets/stylesheets/reviews/_iframe.scss (hunk 2 above)
Comment (ytyeoh) [inline vendor/assets/stylesheets/reviews/_iframe.scss]: add iframe- at in front all become .iframe-XXX

File: vendor/assets/stylesheets/reviews/_iframe.scss (hunk 3 above)
Comment (ytyeoh) [inline vendor/assets/stylesheets/reviews/_iframe.scss]: change become a.red-color:hover,
a.red-color:active,
a.red-color:visited,
a.red-color:focus{
	color: $primary-color;
	text-decoration: none;
}

File: vendor/assets/stylesheets/reviews/_iframe.scss (hunk 4 above)
Comment (ytyeoh) [inline vendor/assets/stylesheets/reviews/_iframe.scss]: fix

File: vendor/assets/stylesheets/reviews/_iframe.scss (hunk 5 above)
Comment (ytyeoh) [inline vendor/assets/stylesheets/reviews/_iframe.scss]: fix

**Pull request closed without merging**